app/javascript/drivers/method.js

```diff
@@ -1,11 +1,32 @@
-import urls from '../urls'
-
-function invokeCommand(element, payload = {}) {
-  const src = payload.src
-  payload = { ...payload }
-  delete payload.src
-  delete payload.href
-  element.setAttribute('href', urls.build(src, payload))
+let activeElement
+let activePayload
+
+const reset = () => {
+  activeElement = null
+  activePayload = null
+}
+
+const invokeCommand = (element, payload = {}) => {
+  activeElement = element
+  activePayload = payload
 }
 
+const ammendForm = form => {
```

Comment:
Did you mean `amendForm`? 
![CleanShot 2024-02-21 at 17 44 27@2x](https://github.com/hopsoft/turbo_boost-commands/assets/106795/564183b6-c315-4de5-9227-9262c2b7834d)